src/conn/conn_reconfig.c

```diff
@@ -152,7 +152,15 @@ __wti_conn_compat_config(WT_SESSION_IMPL *session, const char **cfg, bool reconf
      * don't rewrite the turtle file if there is an error.
      */
     if (reconfig) {
+#ifdef _MSC_VER
```

Comment:
Once this is merged I'll take up WT-14051 to remove the need for all these `#ifdef _MSC_VER` conditionals as it's a quick win. Note I'll just be fixing the Windows compilation, not enabling live restore on Windows machines.